app/assets/javascripts/DropDownMenu/SingleSelectDropDown.js

```diff
@@ -27,15 +27,27 @@ export class SingleSelectDropDown extends React.Component {
         return (
           <ul data-testid={"options"}>
             {options.map(option => {
-              return (
-                <li key={option} onClick={e => this.onSelect(e, option)}>
-                  <span>
-                    {this.props.valueToDisplayName != null
-                      ? this.props.valueToDisplayName[option]
-                      : option}
-                  </span>
-                </li>
-              );
+              if (disabled && disabled.includes(option)) {
+                return (
+                  <li key={option} className={"disabled"}>
```

Comment:
This is good. Also make sure that the cursor is *not* a pointer for disabled options.